ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/winc/cucushift-installer-rehearse-aws-ipi-ovn-winc-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-ovn-winc
+  steps:
+    env:
+      SUB_CHANNEL: stable
+      SUB_INSTALL_NAMESPACE: openshift-windows-machine-config-operator
+      SUB_PACKAGE: windows-machine-config-operator
+      SUB_TARGET_NAMESPACES: '!install'
+      SUB_SOURCE: "qe-app-registry"
+      WINDOWS_NODE_REPLICAS: 2
+    pre:
+    - chain: cucushift-installer-rehearse-aws-ipi-ovn-winc-provision
+    test:
+    - ref: cucushift-installer-check
+    post:
+    - chain: cucushift-installer-rehearse-aws-ipi-deprovision
```

Comment:
I am wondering the `cucushift-installer-rehearse-aws-ipi-deprovision` can also destroy the launched winc nodes?

Reply:
Mostly for sure, when uninstalling the cluster the Windows nodes will also get cleaned up. For example, in Flexy-destroy there is no WINC specific code to clean up nodes. During destroy process everything gets cleaned up.

Reply:
good to know.

Reply:
Windows nodes rely on a machineset too, it's the same as with the Linux workers. They don't need of any extra action to get cleaned up. When uninstalling the cluster the machines get destroyed and with that, the worker nodes. For the cases when we use BYOH nodes, we do it inside the test case and the node is cleaned up in the post-test section.